internal/commands/push/command_test.go

```diff
@@ -476,9 +476,7 @@ Failed to discard the draft created for your deployment
 				for _, line := range []string{
 					`Determining changes
 Creating draft
-Pushing changes
-Deploying draft
-Deployment complete`,
+Pushing changes`,
 					"An error occurred while uploading hosting assets: failed to add /404.html: something bad happened",
 					"An error occurred while uploading hosting assets: failed to add /index.html: something bad happened",
 				} {
```

Comment:
Now that deployment waits for decencies and hosting assets, the "Deploying draft" and "Deployment complete" message will not appear if an error occurs.
Would a separate "Deployment failed" message be appropriate here instead? or is the error message enough by itself?